templates/admin/group/group_property_editor.html.ep

```diff
@@ -116,12 +116,23 @@
             <div class="form-group row">
                 <label for="editor-default-priority" class="col-sm-2 control-label" data-toggle="tooltip" title="Default priority for jobs and job templates created in the group">Default priority</label>
                 <div class="col-sm-10">
-                    <input type="number" class="form-control" id="editor-default-priority" name="default_priority" value="<%= $group->default_priority %>">
+                    <input
+                        type="number" class="form-control"
```

Comment:
Looks a bit odd when looking at it in firefox or chromium: 
![Chromium](https://user-images.githubusercontent.com/229240/44985982-97ab5f00-af82-11e8-8e92-ba908f7fe310.png)
![Firefox](https://user-images.githubusercontent.com/229240/44985983-9843f580-af82-11e8-9072-25649747e666.png)